README.md

```diff
@@ -82,74 +74,66 @@ Drop-In is currently localized for [23 languages](https://github.com/braintree/b
 
 ### 3D Secure + Drop-in
 
-The new Drop-In supports 3D-Secure verification. Assuming you have [3D-Secure configured](https://developers.braintreepayments.com/guides/3d-secure/configuration) for your account, enable it in your client with `DropInRequest.requestThreeDSecureVerification(true)` and set an amount. Then, create a ThreeDSecureRequest() object, setting `ThreeDSecurePostalAddress` and `ThreeDSecureAdditionalInformation` fields where possible; the more fields that are set, the less likely a user will be presented with a challenge. For more information, check our [3D Secure Migration Guide](https://developers.braintreepayments.com/guides/3d-secure/migration/android/v3#getting-ready-for-3ds-2). Make sure to attach this object to the `BTDropInRequest` before use.
-
-```java
-ThreeDSecurePostalAddress billingAddress = new ThreeDSecurePostalAddress()
-    .givenName("Jill")
-    .surname("Doe")
-    .phoneNumber("5551234567")
-    .streetAddress("555 Smith St")
-    .extendedAddress("#2")
-    .locality("Chicago")
-    .region("IL")
-    .postalCode("12345")
-    .countryCodeAlpha2("US");
-
-ThreeDSecureRequest threeDSecureRequest = new ThreeDSecureRequest()
-    .amount("1.00")
-    .versionRequested(ThreeDSecureRequest.VERSION_2)
-    .email("test@email.com")
-    .mobilePhoneNumber("3125551234")
-    .billingAddress(billingAddress)
-    .additionalInformation(additionalInformation);
+The new Drop-In supports 3D-Secure verification. Assuming you have [3D-Secure configured](https://developer.paypal.com/braintree/docs/guides/3d-secure/configuration) for your account, create a ThreeDSecureRequest() object, setting `ThreeDSecurePostalAddress` and `ThreeDSecureAdditionalInformation` fields where possible; the more fields that are set, the less likely a user will be presented with a challenge. For more information, check our [3D Secure Migration Guide](https://developer.paypal.com/braintree/docs/guides/3d-secure/migration/android/v4#getting-ready-for-3ds-2). Make sure to attach this object to the `BTDropInRequest` before use.
 
+```kotlin
+val address = ThreeDSecurePostalAddress()
+address.givenName = "Jill" // ASCII-printable characters required, else will throw a validation error
```

Comment:
Is this note (ASCII-printable characters required, else will throw a validation error) also in our docstrings?

Reply:
No doesn't look like it - good call. This class is in the core SDK, but I'll add that to the doc string there 👍

Reply:
Good catch.  This is the source of a lot of frustration for EU/APAC merchants iirc.  The more we call this out, hopefully the less it will catch folks off guard.